README.md

```diff
@@ -172,6 +174,43 @@ data:
     OVN_LOG_LEVEL=dbg
 ```
 
+### Configuring OVNKubernetes On a Hybrid Cluster
+OVNKubernetes supports a hybrid cluster of both Linux and Windows nodes on x86_64 hosts. The ovn configuration is done as described above. In addition the `hybridOverlayConfig` can be included as follows:
+
+Create the cluster using openshift-install. First get the install-config. Use a convenient directory for the cluster.
+```
+$ openshift-install --dir=MY_CLUSTER create install-config
+```
+Edit the MY_CLUSTER/install-config and change the `networkType:` to OVNKubernetes
+
+Next create the manifests:
+```
+$ openshift-install --dir=MY_CLUSTER create manifests
+$ cp MY_CLUSTER/manifests/cluster-network-02-config.yml MY_CLUSTER/manifests/cluster-network-03-config.yml
+```
+Edit the new file as follows:
+- change first line `apiVersion: config.openshift.io/v1` to `apiVersion: operator.openshift.io/v1`
+- Add the following to the `spec:` section
+
+```yaml
+spec:
+  defaultNetwork:
+    type: OVNKubernetes
+    ovnKubernetesConfig:
+      hybridOverlayConfig:
+        hybridClusterNetwork:
+        - cidr: 10.132.0.0/14
+          hostPrefix: 23
+```
+The hybridClusterNetwork `cidr` and hostPrefix are used when adding windows nodes. This "cidr" must not overlap the ClusterNetwork "cidr" or serviceNetwork "cidr".
```

Comment:
@pecameron I meant more:

This CIDR must not overlap the ClusterNetwork CIDR or ServiceNetwork CIDR.